File: 1/exerc1.rb
#Crie uma função que receba dois números e retorne o maior deles.

puts "Digite um número:"
n1 = gets.chomp().to_i
puts "Digite outro número:"
n2 = gets.chomp().to_i

def maior(n1,n2)
  if n1 > n2
    puts "n1: #{n1} é maior que #{n2}"
  else
    puts "n2: #{n2} é maior que #{n1}"
  end
end

maior(n1,n2)

File: 1/exerc1.py
#Crie uma função que receba dois números e retorne o maior deles.

n1 = int(input('Digite um número: '))
n2 = int(input('Digite outro número: '))

def maior(n1, n2):
    if n1 > n2:
        return print(f'{n1} é maior que {n2}')
    else:
        return print(f'{n2} é maior que {n1}')

maior(n1,n2)